cheerlights/app.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import . "github.com/alexellis/blinkt_go"
```

Comment:
I think I will rename `sysfs` to something else since its the examples we've created that aren't part of the original Pimoroni set.